### PR TITLE
Change term on deposity manager

### PR DESCRIPTION
**Pull request by tmoutinho**

### CODE REVIEW

File: src/providers/ActivityProvider.js [hunk 1]
```diff
@@ -21,14 +21,12 @@ const TIMEOUT_DURATION = 10 * MINUTE
...
-    preStringify: (activity) => ( 
-     {
+    preStringify: (activity) => ({
       ...activity,
       status: activity.status.replace('ACTIVITY_STATUS_', ''),
       type: activity.type,
     }),
-    postParse: (activity) => (
-     {
+    postParse: (activity) => ({
```
Comment (tmoutinho) [inline src/providers/ActivityProvider.js:29]: This is just auto-format.

File: src/hooks/constants.ts [hunk 2]
```diff
@@ -2,11 +2,11 @@ import { ConnectionContext } from '@1hive/connect-react'
 import BigNumber from '@lib/bigNumber'
 
 type TokenType = {
-  decimals: number
+  decimals?: number
```
Comment (tmoutinho) [inline src/hooks/constants.ts:5]: This will fix a ts error on ActionCollateral file

File: src/components/Garden/ProposalDetail/ProposalDetail.js [hunk 3]
```diff
@@ -105,7 +105,7 @@ function ProposalDetail({
     actions.resolveAction(proposal.disputeId)
   }, [actions, proposal])
 
-  const handleShowModal = useCallback(mode => {
+  const handleShowModal = useCallback((mode) => {
```
Comment (tmoutinho) [inline src/components/Garden/ProposalDetail/ProposalDetail.js:108]: This is just auto-format.

File: src/components/Garden/DecisionLoader.tsx [hunk 4]
```diff
@@ -1,9 +1,15 @@
 import React from 'react'
 import DecisionDetail from './DecisionDetail/DecisionDetail'
 import { GardenLoader } from '../Loader'
-import useProposalLogic from '../../logic/proposal-logic'
+import useProposalLogic, {
+  ProposalLogicProps,
+} from '../../logic/proposal-logic'
```
Comment (kamikazebr) [inline src/components/Garden/DecisionLoader.tsx:6]:
```suggestion
import useProposalLogic, { ProposalLogicProps } from '@/logic/proposal-logic'
```

Suggestion of use @ instead ../../ to point to src folder.

**Review by kamikazebr — APPROVED** [review]
Looks Good! I left a comment-suggestion.

File: src/components/Garden/Stake/StakingMovements.js [hunk 5]
```diff
@@ -85,10 +85,13 @@ function StakingMovements({ stakingMovements, token }) {
 
   const [selectedPage, setSelectedPage] = useState(0)
 
-  const handlePageChange = useCallback(page => {
+  const handlePageChange = useCallback((page) => {
     setSelectedPage(page)
   }, [])
 
+  const getProposalType = (disputableAddress) =>
+    disputableAddress === config.voting.id ? 'Vote' : 'Decision'
```
Comment (0xGabi) [inline src/components/Garden/Stake/StakingMovements.js:93]:
```suggestion
    disputableAddress === config.voting.id ? 'Decision' : 'Proposal'
```

Comment (0xGabi) [reply src/components/Garden/Stake/StakingMovements.js:93]: The `decision` actions are the ones that alter the Gardens DNA (use the Disputable Voting Aragon app). The `proposals` are both of type suggestion and funding (when requesting funds).

**Review by 0xGabi — CHANGES_REQUESTED** [review]
Well done! I left a cosmetic change on the naming.

File: src/components/Garden/Stake/StakingMovements.js (hunk 5 above)
Comment (tmoutinho) [reply src/components/Garden/Stake/StakingMovements.js]: Sure Gabi 👍🏼

**Review by 0xGabi — APPROVED** [review]

**Pull request merged**